tox.ini

```diff
@@ -30,6 +30,7 @@ commands=py.test --cov=teuthology --cov-report=term -v {posargs:teuthology/test/
 basepython=python2.7
 
 [testenv:flake8]
+ignore=F632
```

Comment:
I would not use this, but fix the python3 related issues one by one, especially when it is declared that 2.7 will be deprecate in less than a year.

Reply:
@kshtsk can you separate the commit into its own PR? the subject would be clear then, Also, teuthology is not ready for python3 yet, all the tests use python2 environment, I see this as error in flake8 thats doing py3 flake tests in py2 environment, when we actually run in py3 environment it would make sense to look at all the errors.

Reply:
> @kshtsk can you separate the commit into its own PR? the subject would be clear then, Also, teuthology is not ready for python3 yet, all the tests use python2 environment, I see this as error in flake8 thats doing py3 flake tests in py2 environment, when we actually run in py3 environment it would make sense to look at all the errors.

"is" should not be used for python2.7 as well:
```
$ python2.7
Python 2.7.15 (default, Jan 18 2019, 02:15:11) 
[GCC 4.2.1 Compatible Apple LLVM 10.0.0 (clang-1000.11.45.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> a=257
>>> b=257
>>> a==b
True
>>> a is b
False
>>> 

```